cni/network/network.go

```diff
@@ -476,60 +443,74 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 		}
 
 		if resultSecondAdd != nil {
-			result = resultSecondAdd
+			ipamAddResult.ipv4Result = resultSecondAdd
 			return nil
 		}
 	}
 
+	if nwCfg.MultiTenancy {
+		cnsClient, er := cnscli.New(nwCfg.CNSUrl, defaultRequestTimeout)
+		if er != nil {
+			return fmt.Errorf("failed to create cns client for multitenancy %w", er)
+		}
```

Comment:
as we discussed, this code can be moved to case network.AzureCNS: after line 471 and remove this code
cnsClient, er := cnscli.New("", defaultRequestTimeout)

Reply:
discussed offline and will be addressed in a separate pr